woocommerce/woocommerce-android
# Shipping Label Payments: disable editing for non-store owners

Closes #3597, disable editing payments for non-store owners, and display a warning about it.

<img width=400 src="https://user-images.githubusercontent.com/1657201/108698445-6df4a280-7504-11eb-9d33-3f7a3f1fc6fc.png" />

#### Testing

1. Make sure your WCS setup has a payment (or multiple) methods.
2. Create an administrator account in your store.
3. Login to the app using this account.
4. Create an order that satisfies the shipping label creation.
5. Open the order details.
6. Click on "create shipping label"
7. Pass all the steps until you reach the payment details.
8. Click on edit button in the payments step.
9. Confirm that editing the payment methods is disabled, and there is a warning to explain it.
10. Confirm that you can edit the email receipts checkbox.

Update release notes:
